spec/xcpretty/parser_spec.rb

```diff
@@ -405,6 +405,20 @@ module XCPretty
         end
       end
 
+      it "parses uncaught exception errors" do
+        @formatter.should receive(:format_uncaught_exception).with(
+          'testRaisingUncaughtException',
+          'NSInvalidArgumentException',
+          '-[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]',
```

Comment:
Line is too long. [118/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.